app/models/grda_warehouse/warehouse_clients_processed.rb

```diff
@@ -810,6 +812,62 @@ def vispdat_priority_score(client_id)
       end
     end
 
+    def last_intentional_contacts(client_id)
+      open_enrollments = GrdaWarehouse::ServiceHistoryEnrollment.entry.ongoing.
+        where(client_id: @client_ids)
+      @services ||= {}.tap do |h|
+        open_enrollments.
+          joins(:service_history_services, :project).
+          merge(GrdaWarehouse::ServiceHistoryService.service_excluding_extrapolated).
```

Comment:
`service_excluding_extrapolated` only ignores SO extrapolated, I think you also want to limit this to SO and ES NbN since all others will have a day for every day in the enrollment.